src/setup/devnet/index.js

```diff
@@ -343,6 +343,13 @@ export class Devnet {
             obj: this,
             ganache: this.ganache
           })
+
+          for (let i = 0; i < this.totalBorNodes; i++) {
+            await fs.copyFile(
+              path.join(this.config.targetDirectory, 'docker-bor-config.toml'),
+              path.join(this.borDir(i), 'config.toml')
```

Comment:
Will this also work for nodes who chose `cli` instead of `config.toml` ?

Reply:
This will work irrespective of that. We are just creating the `config.toml` file here.

Reply:
Looks to me we are copying the file to the respective node directories. So if a node chose cli, will it still have `docker-bor-config.toml` in `devnet` directory ? If it doesn't copy would fail will it not ?

Reply:
Yes, the toml file will be always there in the devnet directory